### ISSUE TITLE
Make it possible for the user to see a list of who has contributed to a map

### ISSUE THREAD

**Issue by Bortseb** [metamaps/metamaps#855]
Currently, on the explore maps page, I can see the number of contributors on a map card... But can't see the list of actual names. That would be really nice.

Also, it used to be possible to see a list of contributors on the map card inside a map. When I just looked again now, this feature is missing. It would be nice to see a list of contributors in this space as well.

I remember @Connoropolous saying something about why this had changed... But I can't recall why it was.

**Comment by Connoropolous** [comment]
The way to see contributors to a map is to open the filter box.
We converted the dropdown in the map info box to be a 'collaborators'
dropdown, meaning who has edit access. Commons maps don't need a
collaborator dropdown, because all users have access.

Agree that it could be good to see contributors somehow on the map cards

**Comment by devvmh** [comment]
What if we added a little link that popped up a list of contributors here

![screen shot 2016-10-27 at 3 52 44 pm](https://cloud.githubusercontent.com/assets/1393708/19759015/89413f82-9c5d-11e6-84c1-97965b743001.png)

We could convert the info box to React along the way, and add a popup with contributors down there. Make sense?

I propose we bump this to 3.5

**Comment by devvmh** [comment]
@Bortseb are you interested in writing React at all? You and I could find a free time slot and work together to move this one over. I suggest it because this is a pretty simple component but it could really demonstrate how to solve the various requirements using react's methodology

**Comment by Bortseb** [comment]
I know almost nothing about react, but it sounds like it will be important moving forward... So sure! I'd like to learn through this issue. How much time do you think we will need? I guess I should review some basics first? When are you free? Through the week I will likely only be able to find a sporadic hour or two here and there to have a video/audio sync where we can really get moving together... But can likely find a chunk of time on a weekend if given enough advanced notice.

**Comment by Connoropolous** [comment]
The big reason that I support this is, a. for the learning, and b. so that when the time comes to switch up the map info box a little bit it will be a simpler process. 

I don't like the map info box much as it is, I think it's a bad replacement for a more functional map info and settings panel. Ah well! One piece at a time. :)

**Comment by Connoropolous** [comment]
moving this out of 3.0

**Comment by Bortseb** [comment]
@devvmh doing a github notifications sweep, and came across this one. Do I need to do a quick react tutorial or something somewhere to get up to speed enough that we can do this together... I can work through the week around lunch time 12-1pm EST. Is that enough?

**Comment by devvmh** [comment]
Yeah, running through the react tutorial at facebook.github.com/react is a
good idea.  do you want to pick some evenings in the next week or two,  or
is a weekend day better?

I think we should find two 3 hour blocks or one 5 hour block to do it right
